resources/energyplus/ProposedEnergy+.idd

```diff
@@ -15410,7 +15410,6 @@ Exterior:FuelEquipment,
        \required-field
        \type choice
        \key Electricity
-       \key Water
```

Comment:
So before, "Water" and "DistrictHeating" were redundant?

Reply:
Or maybe "Water" was never mapped to?

Reply:
`Exterior:WaterEquipment` is there for a reason. This is a change I'm upstreaming to E+ as well.